traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -271,11 +270,19 @@ func SnapshotOldGUIHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	cdn := inf.Params["cdn"]
-
+	cdnID, ok, err := dbhelpers.GetCDNIDFromName(inf.Tx.Tx, tc.CDNName(cdn))
+	if !ok {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusNotFound, fmt.Errorf("couldn't find CDN ID with name %s", cdn), nil)
+		return
+	}
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, err, nil)
+		return
+	}
```

Comment:
If no rows are returned, the error returned from `dbhelpers.GetCDNIDFromName()`, the error returned is `nil`:
https://github.com/apache/trafficcontrol/blob/ae2ced905d223789c818831ead1c6979c488eaf0/traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go#L518-L520

So this should be a 500-level internal server error, not 400 bad request.